### PR TITLE
Adjust charts

### PR DESCRIPTION
**Pull request by swaterkamp**
**What**:
Add x-axis label to bar chart and total amount of vulnerabilities to pie chart slices.

**Why**:

**How**:

**Checklist**:

- [ ] Tests
- [X] [CHANGELOG](https://github.com/greenbone/pheme/blob/master/CHANGELOG.md) Entry
- [ ] Documentation

### CODE REVIEW

**Review by y0urself — DISMISSED** [review]

**Review by nichtsfrei — APPROVED** [review]

**Comment by y0urself** [comment]
Wait for the release workflow before merging. Please.

**Pull request merged**
